app/views/todos/new.js.haml

```diff
@@ -1 +1,2 @@
+$('.modal-container').append("#{escape_javascript( render partial: 'todo_modal')}");
```

Comment:
@idprietop the `todo_modal` file is missing in the PR, right?

Reply:
I sent the todo_modal file in the previous PR.